src/Raven.Client/Http/RequestExecutor.cs

```diff
@@ -2133,11 +2133,16 @@ public void Dispose()
             }
         }
 
-        public async Task<(int Index, ServerNode Node)> GetRequestedNode(string nodeTag)
+        public async Task<(int Index, ServerNode Node)> GetRequestedNode(string nodeTag, bool checkIfAvailable = false)
```

Comment:
due to compatibility consideration we need to create a completely new overload and mark it with `internal`

Reply:
let's rename `checkIfAvailable` -> `throwIfContainsFailures`

Reply:
Done